### PR TITLE
Russian po updates, typo in automation_time_axis.cc fix

### PR DESCRIPTION
**Pull request by psemiletov**
Russian po updates, typo in automation_time_axis.cc fix
Also I've notice that the "range" term from the source is translated differently in many places (as "Область", "Диапазон", "Выделение") - I think there is a need to translate "range" constantly with one word.

### CODE REVIEW

File: gtk2_ardour/automation_time_axis.cc [hunk 1]
```diff
@@ -174,7 +174,7 @@ AutomationTimeAxisView::AutomationTimeAxisView (
 	   as the longest automation label (see ::automation_state_changed()
 	   below). be sure to include a descender.
 	*/
-	auto_dropdown.set_sizing_text(_("Mgnual"));
```
Comment (x42) [inline gtk2_ardour/automation_time_axis.cc]: This _typo_ is intentional. see the comment above. The string needs to include a descender, at least in English.

"Play" is tallest (due to the descender), "Manual" widest.

Comment (psemiletov) [inline gtk2_ardour/automation_time_axis.cc]: Thanks, I see.

**Comment by x42** [comment]
rebased + changed (reverted the "Mgnual") as Ardour 5.10-391 fe5f00c367

**Comment by x42** [comment]
Oops I forgot to change the commit msg (exclude the automation-change). Sorry.

**Comment by psemiletov** [comment]
Thank you!

**Pull request closed without merging**